src/sentry/tasks/integrations/sync_status_inbound.py

```diff
@@ -38,6 +39,16 @@ def sync_status_inbound(
         Group.objects.update_group_status(
             affected_groups, GroupStatus.RESOLVED, ActivityType.SET_RESOLVED
         )
+
+        for group in affected_groups:
+            analytics.record(
+                "issue.resolved",
+                project_id=group.project.id,
+                default_user_id=organizations[0].get_default_owner().id,
+                organization_id=organization_id,
+                group_id=group.id,
+                resolution_type="with_third_party_app",
+            )
```

Comment:
Third party issue resolution analytic event

Reply:
Do we need a test for this one as well? I only see a test change for `automatic`.